dist/ChartContainer.js

```diff
@@ -54,9 +56,11 @@ function _arrayWithHoles(arr) { if (Array.isArray(arr)) return arr; }
 var propTypes = {
   datasource: _propTypes.default.object.isRequired,
   pan: _propTypes.default.bool,
-  zoom: _propTypes.default.bool,
-  zoomoutLimit: _propTypes.default.number,
-  zoominLimit: _propTypes.default.number,
+  zoom: _propTypes.default.shape({
```

Comment:
Are underscores being used because this is a .js file?

Reply:
@Tristan-Trainual you can ignore this entire file and anything in the /dist directory. It's autogenerated when we compile the library. 

This is react files compiled into js.